src/Microsoft.Identity.Client/MsalServiceException.cs

```diff
@@ -123,7 +132,7 @@ public MsalServiceException(string errorCode, string errorMessage, int statusCod
             Claims = claims;
         }
 
-        private HttpResponse _httpResponse;
+        #endregion
```

Comment:
do we need the region?

Reply:
Regions are an editor feature, no effect on build or runtime. So we definitely don't "need" it, but I like grouping things in regions. 

There are projects that make extensive use of regions - "members", "constructors", "public surface", "private methods" are typical regions. It makes scrolling easier, and you can also glance at how complex a class is by looking at how many items are in the public region (hint: only 1 method and some ctor should ideally be public).

Reply:
I vote for no need for regions

---
In reply to: [298062673](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1236#discussion_r298062673) [](ancestors = 298062673)